site.yml

```diff
@@ -12,6 +12,11 @@
       when:
         - ansible_hostname == lookup('env', 'SPLUNK_SEARCH_HEAD_CAPTAIN_URL') or splunk.role == "splunk_search_head_captain"
 
+    - name: Create defaults dir
+      file:
+        path: "{{config.defaults_dir}}"
+        state: directory
+
```

Comment:
probably should set the owner to ansible forcibly, just to be safe.

Reply:
Right on, that makes sense. Amended my commit to add that in.